Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -176,7 +176,13 @@ public let CodeGenerators: [CodeGenerator] = [
         } else {
             propertyName = b.genPropertyNameForWrite()
         }
-        let value = b.randVar()
+        var propertyType = b.type(ofProperty: propertyName)
+        // TODO unify the .unknown => .anything conversion
+        if propertyType == .unknown {
+            propertyType = .anything
+        }
+        // As we switch to .anything, we can force unwrap here.
```

Comment:
I don't think this is true since you could have some property type that is not currently present in the generated program, or not?
Since this case is somewhat rare (should only occur in HybridFuzzer in some cases), maybe you can just do `generateVariable` if you get nil here? And make a comment that this is ok since it's guaranteed that property types can be generated by `generateVariable`?

Reply:
Ah I think you're right :)